app/controllers/api/v2/hosts_controller.rb

```diff
@@ -238,6 +238,18 @@ def rebuild_config
         end
       end
 
+      api :GET, "/hosts/:id/template/:type", N_("Get host template")
+      param :id, :identifier_dottable, :required => true
+      param :type, String, :desc => N_("Template type")
```

Comment:
Is it worth including valid values in here?  You can either put an array replacing String (e.g. `TemplateKind.all...`), or inside the documentation, e.g. the  :operatingsystem_families string configured in config/initializers/apipie.rb.